OpenOversight/test_data.py

```diff
@@ -9,22 +9,22 @@
 
 def populate():
     """ Populate database with test data"""
-    po1 = models.Officer(last_name='Tinkle', first_name='Ivana', race='BLACK',
+    po1 = models.Officer(last_name='TINKLE', first_name='IVANA', race='BLACK',
```

Comment:
Well.. in the database everything is uppercase. But let me fix the query to make it explicit (may not get to it tonight but will do at some point)